interface/overview.rst

```diff
@@ -496,6 +497,67 @@ For the Rock-on named *Emby server*, for instance, the commands would be:
 
 *This example is for newer Rockstor versions.*
 
+.. _rockons_root_reset:
+
+Reset Rock-on Root
```

Comment:
```suggestion
Reset The Rock-ons Root
```

@Hooverdan96 I prefer this reference as it is the root for all rock-ons, not for a single rock-on. And our pics and other doc references hopefully mostly conform to this arbitrary but important reference. [EDIT] ~~I know a 'the' could be nice but we are titles here so I think this floats :) .~~